v2/response.go

```diff
@@ -0,0 +1,56 @@
+package jsonrpc
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// Error codes generated by a JSONRPC Server
+const (
+	CodeParseError        int = -32700
+	CodeInvalidRequest        = -32600
+	CodeMethodNotFound        = -32601
+	CodeInvalidParameters     = -32602
+	CodeInternalError         = -32603
+	CodeMiscError             = -32001
+)
+
+// Error will return any errors in the JSONRPC format.
+type Error struct {
```

Comment:
As it's a jsonrpc err maybe call it `JsonRPCError`? Error seems very generic and might be shadowing variable in existing namespace (unless it's intentional).